cmd/graphy/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"encoding/csv"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"time"
+)
+
+var usage = `Consumes a csv file in the format: YYYY-MM-DD HH:MM:SS,file,timeseries,(start|done)
+and outputs a new csv to stdout that aggregates the time differences between start/end
+
+Usage:
+  graphy [flags] <csvfilename>
```

Comment:
It's intention was terminal graphs when I first made it. But then I remembered Excel will do that better. Maybe one day it'll graph :p